models/public/index.md

```diff
@@ -289,6 +314,13 @@ The reported metric is collected over the alphanumeric subset of icdar 13 (1015
 | -------------- | -------------- | ------------------------------------------------------ | -------- | ------- | -------- |
 | Resnet-FC      | PyTorch\*      | [text-recognition-resnet-fc](./text-recognition-resnet-fc/README.md) | 90.94% | 40.3704 | 177.9668  |
 
+## Text to Speech
+
+| Model Name     | Implementation | OMZ Model Name                                         | Accuracy | GFlops  | mParams  |
+| -------------- | -------------- | ------------------------------------------------------ | -------- | ------- | -------- |
+| ForwardTacotron | PyTorch\* | [forward-tacotron](./forward-tacotron/README.md) |  | 6.66 | 13.81 |
+| WaveRNN        | PyTorch\*      | [wavernn](./wavernn/README.md) |  | 0.37 | 0.4 |
```

Comment:
For composite models, the GFlops/mParams values should be listed on a per-component basic. You can use `mtcnn` as an example.